labs/lab08-airgap-lite/README.md

```diff
@@ -78,304 +73,357 @@ and
 ${REPLICATED_APP}-lab08-airgap-lite
 ```
 
-### Connecting
+### Connecting to the air gap server
 
-First set your application slug, the public IP of your jump box and your first name:
+1. Set your application slug, the public IP of your jump box, and your first name:
 
-```shell
-export JUMP_BOX_IP=...
-export REPLICATED_APP=... # your app slug
-export FIRST_NAME=... # your first name (lower case)
-```
+  ```shell
+  $ export JUMP_BOX_IP=<ip_address>
+  $ export REPLICATED_APP=<application_slug>
+  $ export FIRST_NAME=<your_first_name> # use lower case
```

Comment:
we have generally had in our style guide to avoid the `$` bash prompt in multiline shell scripts 

https://github.com/replicatedhq/kots.io/blob/master/CONTRIBUTING.md#multi-line-code-snippets-should-not-have-the-bash-prompt